ale_linters/ansible/ansible_lint.vim

```diff
@@ -25,7 +25,12 @@ function! ale_linters#ansible#ansible_lint#Handle(buffer, version, lines) abort
 
     if '>=6.0.0' is# l:version_group
         let l:error_codes = { 'blocker': 'E', 'critical': 'E', 'major': 'W', 'minor': 'W', 'info': 'I' }
-        let l:linter_issues = json_decode(join(a:lines, ''))
```

Comment:
ALE has a JSON decode function that can be used here. Simply replace:

```
json_decode(join(a:lines, ''))
```

with:

```
ale#util#FuzzyJSONDecode(a:lines, [])
```

Reply:
Ok, thanks for that info, I changed it.